scalafix-tests/unit/src/test/scala/scalafix/tests/interfaces/ScalafixSuite.scala

```diff
@@ -0,0 +1,87 @@
+package scalafix.tests.interfaces
+
+import java.net.URL
+import java.nio.file.{Files, Path}
+
+import coursierapi.Repository
+import org.scalatest.FunSuite
+import scalafix.interfaces.{Scalafix, ScalafixDiagnostic, ScalafixMainCallback}
+import scalafix.internal.tests.utils.SkipWindows
+
+import scala.jdk.CollectionConverters._
+
+/**
+ * Tests in this suite require scalafix-cli & its dependencies to be published (`sbt +publishLocal`) so that
+ * Coursier can fetch them.
+ */
+class ScalafixSuite extends FunSuite {
+
+  def tmpFile(prefix: String, suffix: String)(content: String): Path = {
+    val path = Files.createTempFile(prefix, suffix)
+    Files.write(path, content.getBytes)
+    path
+  }
+
+  def fetchAndLoad(scalaVersion: String): Unit = {
+    test(s"fetch & load instance for Scala version $scalaVersion", SkipWindows) {
```

Comment:
`error: Unknown rule 'file:C:\Users\RUNNER~1\AppData\Local\Temp\CaptureScalaVersion3780416901127757866.scala'` on Windows... I saw that source-compilation was tagged `SkipWindows` in `ToolClasspathSuite` as well, so I didn't look further.